constants/appLinks.ts

```diff
@@ -77,6 +77,23 @@ const TEXTBOOK_DEMO_START_LEARNING: GeneralLink = {
   }
 }
 
+const TEXTBOOK_DEMO_INTRODUCTION_COURSE_EXTERNAL_RECOMMENDED_READINGS: GeneralLink[] = [
```

Comment:
Please, move this definition to the `introduction-couse.vue` page

Reply:
Sure, as discussed offline, we'll start moving the links to the components that use them. 👌